dev/conductor/ui/test/widget_test.dart

```diff
@@ -27,9 +27,9 @@ void main() {
       );
       await tester.pumpWidget(MyApp(state));
 
-      expect(find.text('Flutter Conductor'), findsOneWidget);
-      expect(find.textContaining(channelName), findsOneWidget);
+      expect(find.textContaining('Flutter Desktop Conductor'), findsOneWidget);
+      expect(find.textContaining('Conductor version'), findsOneWidget);
       expect(find.text('1'), findsNothing);
     });
-  }, skip: Platform.isWindows); // This app does not support Windows [intended]
+  }, skip: kIsWeb || (!kIsWeb && Platform.isWindows)); // This app does not support web or Windows [intended]
```

Comment:
we don't run CI tests on the web, this is not necessary

Reply:
for more context, test files run on the HOST device, not the TARGET. we DO have web tests, but those are run from either a Linux, Windows or Mac host, and they then compile web target code.